src/include/planner/csv_scan_plan.h

```diff
@@ -61,6 +61,8 @@ class CSVScanPlan : public AbstractScan {
 
   const std::string &GetFileName() const { return file_name_; }
 
+  const std::string GetInfo() const override { return "CSVScan"; }
```

Comment:
Maybe add "plan". Providing concrete information for each field would be even better (I realize the other plans don't do that either, so it's up to you)